x-pack/plugins/security_solution/public/common/components/event_details/overview/index.test.tsx

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Overview } from './';
+import { TestProviders } from '../../../../common/mock';
+
+jest.mock('../../../lib/kibana');
+
+describe('Event Details Overview Cards', () => {
+  it('renders all cards', () => {
+    const { getByText } = render(
+      <TestProviders>
+        <Overview {...props} />
+      </TestProviders>
+    );
+
+    getByText('Status');
+    getByText('Severity');
+    getByText('Risk Score');
+    getByText('Rule');
+  });
+
+  it('renders all cards it has data for', () => {
+    const dataWithoutRiskScore = props.data.filter(
+      (data) => data.field !== 'kibana.alert.rule.risk_score'
+    );
+
+    const fieldsWithoutRiskScore = {
+      'kibana.alert.rule.severity': props.browserFields.kibana.fields['kibana.alert.rule.severity'],
```

Comment:
It might exist already, but if not we should create a constants file for all these `kibana.alert.*` fields. Not necessary for this PR, but we should try to do it for other PR's moving forward

Reply:
Yep, great idea. I did not find one but then I might just not know the codebase well enough ;)

Reply:
Yea, no worries.